File: csharp/Base60.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace BaseN
{
    /// <summary>
    /// NewBase60 base converter, see http://tantek.pbworks.com/w/page/19402946/NewBase60
    /// </summary>
    public class Base60 : BaseNConverter
    {
        public const string ALPHABET = "0123456789ABCDEFGHJKLMNPQRSTUVWXYZ_abcdefghijkmnopqrstuvwxyz";

        public Base60()
            : base(ALPHABET)
        {
        }

    }
}

File: java/base_n/Base60.java
package base_n;

/**
 * NewBase60 base converter, see http://tantek.pbworks.com/w/page/19402946/NewBase60
 * @author	jcoder
 */
public class Base60 extends BaseNConverter
{
    public static final String ALPHABET = "0123456789ABCDEFGHJKLMNPQRSTUVWXYZ_abcdefghijkmnopqrstuvwxyz";

    public Base60() {
        super(ALPHABET);
    }

}
